src/wsdl/elements.ts

```diff
@@ -225,13 +225,22 @@ export class ElementElement extends Element {
     let type: any = this.$type || this.$ref;
     if (type) {
       type = splitQName(type);
+      const typeStorage = this.$type ? definitions.descriptions.types : definitions.descriptions.elements;
       const typeName: string = type.name;
+      if(typeName in typeStorage){// typeにtypeNameが入ってしまった場合の対応、本来はこのようなことが起きないように対処すべきかもしれない。
+        if (this.$ref) {
+          element = typeStorage[typeName];
```

Comment:
Probably, need to check that `typeStorage[typeName]` has a value, could be undefined, null ... anything